src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.InProcess.cs

```diff
@@ -116,6 +105,7 @@ private static string GetItemKind(DeclaredSymbolInfo declaredSymbolInfo)
                 case DeclaredSymbolInfoKind.Interface:
                     return NavigateToItemKind.Interface;
                 case DeclaredSymbolInfoKind.Constructor:
+                case DeclaredSymbolInfoKind.ExtensionMethod:
```

Comment:
Previously, we would pick the glyph based on the *symbol*.  Thus, we could pick 'Extension Method' as the glyph. Now that i'm moving to storing the information on DeclaredSymbolInfo i needed to be able to store that a method was actually an extension method.  However, once i did that, i needed to update this switch statement so it would not hit the defualt case and throw.